tests/click_tests/test_autolock.py

```diff
@@ -132,10 +133,10 @@ def test_autolock_interrupts_passphrase(device_handler: "BackgroundDeviceHandler
     assert layout.text == "< PassphraseKeyboard >"
 
     CENTER_BUTTON = buttons.grid35(1, 2)
-    # total sleep time: 5 * 1.1 = 5.5 seconds
-    for _ in range(5):
+    # keep clicking for long enough to trigger the autolock if it incorrectly ignored key presses
```

Comment:
i don't think this is true? not sure what the original intention was with the `5` here.

obviously the first test case is checking that the passphrase keyboard keeps Trezor awake
and this other one is checking that if you auto-lock while the passphrase keyboard is up, it gets correctly cancelled.
why enter letters in the first place, i'm not sure. maybe something like, "even if you started typing, autolock must kill it"?
in which case the number of presses doesn't seem important

Reply:
[84bfd40](https://github.com/trezor/trezor-firmware/pull/2806/commits/84bfd40b2fad843cda88447746dc17775cd23d6a)